pkg/dom0-ztools/rootfs/etc/init.d/000-mod-params

```diff
@@ -36,3 +36,41 @@ zfs_set_default_parameters() {
 # Load ZFS and set parameters
 modprobe zfs
 zfs_set_default_parameters
+
+if ! test -f /proc/vmcore; then
+    #
+    # Load other modules for regular EVE kernel (not crashdump).
+    #
+    # FIXME: This was moved from the images/rootfs.yml.in, thus preserves
+    # FIXME: original indifference to any errors, which can happen while
+    # FIXME: loading on a machine, where particular hardware is missing.
+    # FIXME:
+    # FIXME: Future editor of this list, please take the trouble and at
+    # FIXME: least describe for what hardware and under what conditions
+    # FIXME: the module should be loaded.
+
+    modprobe nct6775              2>/dev/null
+    modprobe w83627hf_wdt         2>/dev/null
```

Comment:
No directly related to your restructuring but ... For how long time can the crash kernel run? Do we need to run the watchdog process and have the watchdog drivers?

Reply:
We don't load those modules for the crashkernel, see the comment:
```
    # Load other modules for regular EVE kernel (not crashdump).
   ```